code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1057,6 +1058,21 @@
 	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
 	..()
+/datum/reagent/medicine/penthrite
+	// Same as corazone. but slightly better
+	name = "Penthrite"
+	description = "An explosive compound used to treat certain heart conditions."
+	color = "#F5F5F5"
+	self_consuming = TRUE
+
+/datum/reagent/medicine/penthrite/on_mob_metabolize(mob/living/M)
+	..()
+	M.adjustOrganLoss(ORGAN_SLOT_HEART,-0.15)
```

Comment:
@optimumtact did mention that there should be no organ healing chems, but at this point so many have snuck through it's impossible to police